addons/xeh_a3/CfgEventHandlers.hpp

```diff
@@ -1,249 +0,0 @@
-                SLX_BIS = "(_this select 0) execVM ""\A3\weapons_f\data\scripts\smoke.sqf"";";
-        };
-        class B_AssaultPack_khk_holder /* : WeaponHolder */ {
-                SLX_BIS = "(_this select 0) addBackpackCargo [""B_AssaultPack_khk"",1];";
-        };
-        class Underwear_F /* : B_Soldier_F */ {
-                SLX_BIS = "(_this select 0) execVM ""\A3\Characters_F\Common\scripts\randomize.sqf""";
-        };
-        class FirePlace_burning_F /* : Land_FirePlace_F */ {
-                SLX_BIS = "(_this select 0) inflame true";
-        };
-        class Heli_Light_01_civil_base_F /* : Heli_Light_01_base_F */ {
-                SLX_BIS = "if (local (_this select 0)) then {[(_this select 0), """", [], false] call bis_fnc_initVehicle;};";
-        };
-        class Static_Designator_01_base_F /* : StaticWeapon */ {
-                SLX_BIS = "(_this select 0) execVM ""\A3\Static_F_Mark\Designator_01\Scripts\init.sqf""";
-        };
-        class Static_Designator_02_base_F /* : StaticWeapon */ {
-                SLX_BIS = "(_this select 0) execVM ""\A3\Static_F_Mark\Designator_02\Scripts\init.sqf""";
-        };
-};
-class Extended_killed_Eventhandlers {
-        class Helicopter /* : Air */ {
-                SLX_BIS = "_this call (uinamespace getvariable 'BIS_fnc_effectKilled')";
-        };
-        class Plane /* : Air */ {
-                SLX_BIS = "_this call (uinamespace getvariable 'BIS_fnc_effectKilled')";
-        };
-        class Car_F /* : Car */ {
-                SLX_BIS = "_this call (uinamespace getvariable 'BIS_fnc_effectKilled')";
-        };
-        class Ship_F /* : Ship */ {
-                SLX_BIS = "_this call (uinamespace getvariable 'BIS_fnc_effectKilled')";
-        };
-        class Boat_Civil_01_base_F {
-                SLX_BIS = "_this select 0 animate [""HidePoliceSigns"",1]; _this select 0 animate [""HideRescueSigns"",1]; _this select 0 animate [""HidePolice"",1];";
-        };
-        class Tank_F /* : Tank */ {
-                SLX_BIS = "_this call (uinamespace getvariable 'BIS_fnc_effectKilled')";
-        };
-        class I_G_Offroad_01_F /* : Offroad_01_base_F */ {
-                SLX_BIS = "_this call (uinamespace getvariable 'BIS_fnc_effectKilled'); _this select 0 animate [""DamageUnHideConstruction"",0];";
-        };
-        class I_G_Offroad_01_armed_F /* : Offroad_01_armed_base_F */ {
-                SLX_BIS = "_this call (uinamespace getvariable 'BIS_fnc_effectKilled'); _this select 0 animate [""DamageUnHideConstruction"",0];";
-        };
-        class Land_Communication_anchor_F /* : House_F */ {
-                SLX_BIS = "[(_this select 0)] execVM ""\A3\Structures_F\Ind\Transmitter_Tower\Scripts\anchor_ruins.sqf""";
-        };
-        class Land_Communication_F /* : House_F */ {
-                SLX_BIS = "[(_this select 0)] execVM ""\A3\Structures_F\Ind\Transmitter_Tower\Scripts\tower_ruins.sqf""";
-        };
-};

```

Comment:
It's no longer overwritten, since I use a sub class (new in v1.54) for the XEH events.